src/goto-checker/solver_factory.cpp

```diff
@@ -213,20 +214,148 @@ make_satcheck_prop(message_handlert &message_handler, const optionst &options)
   return satcheck;
 }
 
-std::unique_ptr<solver_factoryt::solvert> solver_factoryt::get_default()
```

Comment:
May I suggest to retain the name of this method. We may wish to change the default solver to be something that's not a SAT solver one day.

Reply:
Done